### ISSUE TITLE
Error! Binaries Corrupted/Missing.

### ISSUE THREAD

**Issue by Zerem-12** [StefanLobbenmeier/youtube-dl-gui#158]
**Describe the bug**
Self-explanatory. It gives me Binaries Corrupted/Missing.

**To Reproduce**
1. Use this YouTube link: https://www.youtube.com/watch?v=YFUUfNCHUEM
2. Get its cookies accordingly.
3. Error happens, even if you disable antiviruses such as Windows Defender.

**Screenshots**
<img width="633" alt="Error! Binaries Missing-Corrupted  Issue #1" src="https://github.com/user-attachments/assets/ad1be064-5f3c-49b3-a4a7-e2932c6ec135" />
<img width="635" alt="image" src="https://github.com/user-attachments/assets/29594668-1f38-4201-bff7-5629d4135db6" />

**Comment by StefanLobbenmeier** [comment]
You are saying this only happens when you attempt to add cookies?

I wonder if this is somehow related to a windows privacy settings preventing applications from reading from your one drive or something like that

**Comment by Mendungo** [comment]
Any fix yet? Happening to me too

**Comment by StefanLobbenmeier** [comment]
Do you reproduce this only with cookies? And application version is 2.5.4?

**Comment by Zerem-12** [comment]
![Image](https://github.com/user-attachments/assets/77aff3ae-6785-4c06-8604-8f7164c81527)

I got version 2.5.4 and reproduced the steps above; now it just gives me "Error! Unhandled Error."

**Comment by Zerem-12** [comment]
![Image](https://github.com/user-attachments/assets/d86dab2c-d505-4385-ac96-e2dbe52c3037)

Attempting to try again also changes the link to "undefined" and tells you that the URL is not supported.

**Comment by StefanLobbenmeier** [comment]
> Attempting to try again also changes the link to "undefined" and tells you that the URL is not supported.

That’s another issue, it’s everytime you retry a failed item the url is „undefined“

So the only difference is cookies file right? Can you try one more thing which is to move the cookies file from a path like C:/Users/… to either C:/cookies.txt or a USB stick or something like that to confirm my suspicion about user folder being the issue?

**Comment by Zerem-12** [comment]
Used a USB; still a fail.

But I did notice that the C:/ is instead a D:\ now.

![Image](https://github.com/user-attachments/assets/04f43a22-5244-4b57-a8d6-e3f933ca05fe)

Below is the USB in question.

![Image](https://github.com/user-attachments/assets/0ece3065-bb89-4ce8-b1be-7e73d625995f)

**Comment by arielprenthuis** [comment]
I'm in the U.S. and I have the same issue, disabled all windows security.  I downloaded the cookies.txt externsion, did the same as above individual same result.  binaries missing.

**Comment by StefanLobbenmeier** [comment]
I had another suspicion that it could be a space in the path name but there is no space in the screenshot so should be good.

\ instead of / is correct on windows afaik

Could you try if it works in v3? https://github.com/StefanLobbenmeier/yt-dlp-compose/releases/tag/3.0.44

You can try getting cookies from browser directly or from file

**Comment by StefanLobbenmeier** [comment]
> the cookies.txt externsion

Do not use that extension anymore: #149

**Comment by Zerem-12** [comment]
> I had another suspicion that it could be a space in the path name but there is no space in the screenshot so should be good.
> 
> \ instead of / is correct on windows afaik
> 
> Could you try if it works in v3? https://github.com/StefanLobbenmeier/yt-dlp-compose/releases/tag/3.0.44
> 
> You can try getting cookies from browser directly or from file

Was I supposed to download the .exe file and place it in the open-video-downloader folder?
Because if not then I bet I'm doing this wrong lol

![Image](https://github.com/user-attachments/assets/cf1a4d44-67fe-4f9d-83c1-4e8729928690)

It still gave me the error though so this is probably not the correct way to do it xd

**Comment by StefanLobbenmeier** [comment]
You should be able to just open the application at-dlp-compose.exe , it’s independent from v2

**Comment by Zerem-12** [comment]
> You should be able to just open the application at-dlp-compose-3.0.44.exe , it’s independent from v2

Not sure what to do here:
![Image](https://github.com/user-attachments/assets/e44b6327-ad7c-4ce7-b292-db3d1323849b)

**Comment by StefanLobbenmeier** [comment]
Actually on Windows 10 it seems to not add it to the start menu, but you can find it in C:\Program Files\yt-dlp-compose

Not sure why it’s not added to the menu, will have to check if there is a way to do that

**Comment by Mendungo** [comment]
So is this fixed yet?

**Comment by StefanLobbenmeier** [comment]
Yeah fixed in v3 - you can try that installer and it should create a start menu item and a shortcut on the desktop to launch the app

https://github.com/StefanLobbenmeier/yt-dlp-compose/releases/tag/3.0.45

Regarding fixing this in v2 I am still unsure what is causing it, so likely no fix for it there.

**Comment by Zerem-12** [comment]
So I can say that the app looks different
...What do I put here though lol

![Image](https://github.com/user-attachments/assets/31fe160e-5181-4c12-9acf-fcce6e414a2b)

**Comment by Zerem-12** [comment]
At least it's not giving me an instant error though lol

![Image](https://github.com/user-attachments/assets/e0a6d1a9-01d1-41e9-bf7f-addf63d1a9b6)

Also sorry I'm slow lmao

**Comment by Mendungo** [comment]
unfortunately v3 doesn't have the audio only version, in mp3 format o better. it downloads only videos in mk4 format which is not what some people including myself are looking for

**Comment by StefanLobbenmeier** [comment]
Hmm I will need to add better error reporting to figure out why it’s stuck there, so you can see the logs. It’s next on the list
